webonary-cloud-api/lambda/mongo.ts

```diff
@@ -5,59 +5,23 @@ let cachedDb: MongoClient;
 export const DB_URL = process.env.DB_URL as string;
 export const DB_NAME = process.env.DB_NAME as string;
 export const COLLECTION_ENTRIES = 'webonaryEntries';
+export const DB_MAX_UPDATES_PER_CALL = 50;
 
 export interface DbFindParameters {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
   [key: string]: any;
 }
 
 export async function connectToDB(): Promise<MongoClient> {
-  console.log('=> connect to database');
-
   if (cachedDb) {
-    // console.log('=> using cached database instance');
-    // return Promise.resolve(cachedDb);
+    return Promise.resolve(cachedDb);
```

Comment:
I initially had problems with caching, but got it fixed while I was doing refactoring. 
The key was not closing the db connection at the end of Lambda execution in loadEntry.ts